### DISCUSSION TITLE
How can to make a customization on "." popup at the bottom row?

### DISCUSSION THREAD

**Discussion by exampleex** [florisboard/florisboard#2098] [Q&A] [answered]
Sorry for noob question, where can i make a customization "." key pop up (left side of enter key)? 

![IMG_20221026_194407](https://user-images.githubusercontent.com/21957372/198030013-ec8611fb-7b27-4849-8573-932a595967bc.jpg)

**Comment by patrickgold** [comment, marked as answer]
At the moment you can't I am sorry, for this you have to wait until either I or someone else implements runtime keyboard layout customization (#196).

**Reply by exampleex** [reply]
Thank you for your answer.
